site/ipam/troubleshooting-ipam.md

```diff
@@ -43,3 +43,12 @@ the total allocation range and default subnet. Columns are as follows:
       for more
 * 'Range' - total allocation range set by `--ipalloc-range`
 * 'DefaultSubnet' - default subnet set by `--ipalloc-default-subnet`
+
+Per-peer allocation statistics and reachability can be obtained with
```

Comment:
Calling this "*allocation* statistics" is misleading. We, quite consistently, refer to "allocations" when talking about assigning IPs to containers, not when dividing the IP range between peers, which, iirc, is what the figures here refer to. In ipam.md#persistence we refer to this information as "The division of the IP allocation range amongst peers".

Also the "unreachable!" deserves elaboration and reference to the docs for `rmpeer`.

Generally, we need to think about what a "good" and "bad" output would look like here, and guide the user accordingly.